### ISSUE TITLE
Usersnap Feedback - Do we know who "fidele192" is and why they keep adding empty entries? They [...]

### ISSUE THREAD

**Issue by paninee** [participedia/usersnaps#1419]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: Do we know who "fidele192" is and why they keep adding empty entries? They made 5 more earlier today, in addition to several from 2 days ago. I usually hide these types of entries but I think Jesi & Pan might be working on a solution.
[Open #1102 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/1102)

<a href='https://usersnappublic.s3.amazonaws.com/2021-07-30/15-26/160e35fa-5499-4632-8930-0e6a7d991fe7.png'>![Usersnap Feedback - Do we know who "fidele192" is and why they keep adding empty entries? They (...)](https://usersnappublic.s3.amazonaws.com/2021-07-30/15-26/933acea3-caf5-4988-880e-9244bf16f956.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2021-07-30/15-26/160e35fa-5499-4632-8930-0e6a7d991fe7.png'>Download original image</a>

**Browser**: Firefox 90 (Windows 10)
**Referer**: [https://participedia.net/search?page=1](https://participedia.net/search?page=1)
**Screen size**: 1760 x 1100 **Browser size**: 1760 x 939

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

--

A friendly reminder from Usersnap 👋

There’s a new widget in town. Make your bug reporting process sleeker and more advanced by changing to the new project template.

Learn more about how to transfer your project (and what new features you’ve been missing out on) in this [blog post](https://usersnap.com/blog/platform-new-widget/).

**Comment by jesicarson** [comment]
I don't know who they are, no. @paninee can you look into this asap? Thank you!

**Comment by jesicarson** [comment]
I'll email them now and cc you Pan. We may need to block them for now.

**Comment by jesicarson** [comment]
@plscully fyi I looked closer at this and realized that the entries look like tests. Turns out this is Pan’s test account! She will look at this and make sure the tests are removed from the database. I’m leaving them for now because there might be an issue with removing the entries that Pan needs to review. She’s back from holidays this week. Anyway, thanks for flagging! Just thought you might want an update :)
